lib/tasks/roles.rake

```diff
@@ -0,0 +1,49 @@
+desc 'Add application-wide admin privileges to a user'
+task admin: :environment do
+  puts 'Creating an admin user.'
+  u = prompt_to_create_user
+  Role.create(user: u, resource: nil, role: 'admin')
+  puts 'User created.'
+end
+
```

Comment:
Could you add some info to the README so that people know they can use these rake tasks to created admin users and presses.